web/libs/ui/src/lib/label/label.tsx

```diff
@@ -0,0 +1,31 @@
+import { forwardRef } from "react";
```

Comment:
since Label has been unified here - we should look at updating all instances of Label usage to leverage this

Reply:
we cant actually do this now - the visuals for Label used with non-Toggle components are different - we need to standardize that a bit more which is going to involve more work which is out of scope for this ticket